### ISSUE TITLE
Server categories feedback thread

### ISSUE THREAD

**Issue by drtshock** [Minehut/Meta#627]
We just added a way to tag servers with categories on your dashboard. Please use this thread to suggest any category changes (adding or removing any categories).

Initial list:
- SMP
- Factions
- Prison
- Skyblock
- Gens
- Minigames
- Farming
- Creative
- 1.8 PVP
- PVP
- Roleplay
- RPG

**Comment by BurningBrimstone** [comment]
* Skymining
* Box

**Comment by drtshock** [comment]
How is sky mining different from gens?

**Comment by killergotrekt** [comment]
Idle

**Comment by BurningBrimstone** [comment]
> 
> 
> How is sky mining different from gens?

Gens typically refers to "generators" that spawn items above them after a certain interval, seen accompanied by plots or a pvp arena. Sky mining refers to cubes of resources on islands that people mine out.

**Comment by Kastor07** [comment]
PVE
Plots

**Comment by drtshock** [comment]
> PVE
> Plots

Do we need both creative and plots? And is PVE not covered by the other categories up there?

**Comment by Kastor07** [comment]
> > PVE
> > Plots
> 
> Do we need both creative and plots? And is PVE not covered by the other categories up there?

sure, its technically covered but dont you want users to be very specific so the players can get exactly what they want?

**Comment by killergotrekt** [comment]
RPG is pretty much PVP and PVE

**Comment by BurningBrimstone** [comment]
If we're doing specifics;
* Kit PvP

**Comment by Kastor07** [comment]
oh true!

**Comment by Kastor07** [comment]
also 1.9 pvp could be a tag

**Comment by BurningBrimstone** [comment]
> 
> 
> also 1.9 pvp could be a tag

Thats... just PvP

**Comment by B4ssouli** [comment]
Dupe

**Comment by BurningBrimstone** [comment]
* Skymining
* Box
* Resource pack
* Kit PvP
* Dupe
* Anarchy
* Parkour

**Comment by CerialPvP** [comment]
Arcade or minigames

**Comment by MrEnxo** [comment]
Custom Gamemode

**Comment by gamerboiuy** [comment]
Dupe

**Comment by iGalaxyYT** [comment]
> Arcade or minigames

Minigames is already a category.

> Custom models (Optifine)

This can be included in the "Resource Pack" category that you suggested. Besides, you really shouldn't be making use of OptiFine for server resource packs with all the capabilities that Vanilla has now.

**Comment by nohyphens7** [comment]
Anarchy - Outside of me owning 2 different anarchy servers and how this would personally benefit me, but I would say it is important for players to be able to search for servers specifically that have no rules and no restrictions for then if players wouldn't want to be in an environment with that sort of "lawlessness" they would specifically avoid those servers in the category and same principle vice versa.

**Comment by quadmoo** [comment]
Add the tags 'Theme Park' and 'Elytra Quest'

(Elytra Quest is for servers that are not Elytra Mine, but have quests and elytra based challenges, I miss these servers and so it would help if I could see all that are online if any are)

**Comment by Prorickey** [comment]
Bedrock Box or Box

**Comment by ahoulton** [comment]
If we are gonna add 1.8 PvP, maybe also add 1.9 PvP.

Also, what about a Development and Testing tag(s) for the servers made for those reasons?

**Comment by iGalaxyYT** [comment]
> If we are gonna add 1.8 PvP, maybe also add 1.9 PvP.

It was already said above that the "PvP" tag is for 1.9+ PvP

**Comment by ahoulton** [comment]
> It was already said above that the "PvP" tag is for 1.9+ PvP

Didn't see that, removed it from the original comment.

**Comment by drtshock** [comment]
> What about a Development and Testing tag(s) for the servers made for those reasons?

I imagine those servers won't have any tags since they won't be publicly visible.

**Comment by CerialPvP** [comment]
> Bedrock Box or Box

Some box servers don't use a bedrock box so let's just do Box.

**Comment by CerialPvP** [comment]
> Anarchy - Outside of me owning 2 different anarchy servers and how this would personally benefit me, but I would say it is important for players to be able to search for servers specifically that have no rules and no restrictions for then if players wouldn't want to be in an environment with that sort of "lawlessness" they would specifically avoid those servers in the category and same principle vice versa.

There are not many anarchy servers on MH but that is a great idea

**Comment by MrEnxo** [comment]
> > What about a Development and Testing tag(s) for the servers made for those reasons?
> 
> I imagine those servers won't have any tags since they won't be publicly visible.

but what if its like a public testing server thats in beta or some stuff like thatq

**Comment by CerialPvP** [comment]
Maybe Roleplay tag for roleplaying servers

**Comment by quick007** [comment]
Add MMO (some RPG's are meant to be singleplayer experiences/where you can see but not really interact w/other players)

**Comment by Chooofy** [comment]
Here are some categories I suggest.

1. Sci-Fi
2. Open World
3. Magical
4. Lore

**Comment by iGalaxyYT** [comment]
> Here are some categories I suggest.
> 
> 1. Sci-Fi
> 2. Open World
> 3. Magical
> 4. Lore

👍 These should definitely be added. More categories related to the server theme are needed, rather than just the genre/server type.

**Comment by Chooofy** [comment]
Some more categories

1. Grinding
2. Mythical
3. Medieval
4. Futuristic

**Comment by Chooofy** [comment]
Forgot to mention

1. Play with Friends

**Comment by OGContent** [comment]
Clicker game

**Comment by quick007** [comment]
> Clicker game

@lj-pixel

**Comment by lj-pixel** [comment]
Please don't remind me of my past failures.... but yes clicker tag should be a thing too!> > Clicker game
> 
> 
> 
> @lj-pixel

**Comment by CerialPvP** [comment]
Add events tag for event servers

**Comment by MiginsMC** [comment]
> Add events tag for event servers like MasterEvnt or MHWeekends or other event servers out there

Those would have to have some sort of restriction. Could possibly be manually added via the admin panel

**Comment by ahoulton** [comment]
> 
> 
> > Add events tag for event servers like MasterEvnt or MHWeekends or other event servers out there
> 
> Those would have to have some sort of restriction. Could possibly be manually added via the admin panel

Why would these need a restriction? Maybe an "Approved Events" tag if you are worried about scam servers.

**Comment by Oiiink** [comment]
> Add events tag for event servers like MasterEvnt or MHWeekends or other event servers out there
I agree with this, I own an event server and it would come in handy for an event tag.

**Comment by ikeyIX** [comment]
Survival, Economy and Events

**Comment by colton-boi** [comment]
Puzzle
